packages/react-components/react-tag-picker/stories/src/TagPicker/TagPickerAppearance.stories.tsx

```diff
@@ -28,11 +29,18 @@ const Example = ({ appearance }: Pick<TagPickerProps, 'appearance'>) => {
   };
   const tagPickerOptions = options.filter(option => !selectedOptions.includes(option));
 
+  const labelColor =
+    appearance === 'filled-lighter' || appearance === 'filled-darker'
+      ? tokens.colorNeutralForegroundInverted2
+      : tokens.colorNeutralForeground1;
+
+  const label = <Label style={{ color: labelColor }}>Select Employees</Label>;
```

Comment:
```suggestion
  const label = <span style={{ color: labelColor }}>Select Employees</span>;
```
small nit that I just noticed -- the label slot of Field is already a `<Label>` component, so passing in another `<Label>` as a child will result in nested `<label>` elements.

That doesn't really cause any practical problems, but it is a bit weird 😅